modules/xhbBidAdapter.js

```diff
@@ -16,7 +16,7 @@ const XhbAdapter = function XhbAdapter() {
       {
         key: 'hb_xhb_deal',
         val: function (bidResponse) {
-          return bidResponse.adId;
+          return bidResponse.dealId;
```

Comment:
@mkendall07 If an ad is returned, the adapter sets the deal ID to a hardcoded value (see line 133), so I assume this should be fine.